velox/docs/monthly-updates/august-2022.rst

```diff
@@ -0,0 +1,80 @@
+******************
+August 2022 Update
+******************
+
+Documentation
+=============
+
+* Add documentation for :func:`combinations` Presto function.
+* Add documentation for :doc:`/functions/window`.
+* Add documentation for :func:`array_sort`.
```

Comment:
Let's combine this with documentation for 'combinations' function and move to the end of the list.

```
Add documentation for :func:`combinations` and :func:`array_sort` Presto functions.
```